admin/acceptance_tests/conftest.py

```diff
@@ -29,7 +39,10 @@ def dbsession(settings):
 def transact(dbsession):
     t = dbsession.begin_nested()
     yield
-    t.rollback()
+    try:
```

Comment:
this should not happen, if it does, it means there is an error else where.
at least, should we add an enormous log one cannot not see ?